src/makeCalculator.js

```diff
@@ -36,8 +36,61 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(figure) {
+      this.result += figure;
+
+      return this;
+    },
+
+    subtract(figure) {
+      this.result -= figure;
+
+      return this;
+    },
+
+    multiply(figure) {
+      this.result *= figure;
+
+      return this;
+    },
+
+    divide(figure) {
+      this.result /= figure;
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(value, number) {
```

Comment:
<img width="556" alt="image" src="https://github.com/mate-academy/js_advanced_calculator/assets/70220667/fefbd260-1bc7-46d1-9535-dfc9b5aec9c1">
you need to take the callback as the parameter
<img width="335" alt="image" src="https://github.com/mate-academy/js_advanced_calculator/assets/70220667/8ee7d6ec-5abc-468d-bdd3-85ca7b84c935">